lib/measures/CreateTypicalBuilding/resources/create_typical_resources.rb

```diff
@@ -0,0 +1,173 @@
+module CreateTypicalBldgConstants
+
+  CLIMATE_ZONES = ['Lookup From Model',  'ASHRAE 169-2013-1A', 'ASHRAE 169-2013-2A', 'ASHRAE 169-2013-2B',
+                   'ASHRAE 169-2013-3A', 'ASHRAE 169-2013-3B', 'ASHRAE 169-2013-3C', 'ASHRAE 169-2013-4A',
+                   'ASHRAE 169-2013-4B', 'ASHRAE 169-2013-4C', 'ASHRAE 169-2013-5A', 'ASHRAE 169-2013-5B',
+                   'ASHRAE 169-2013-6A', 'ASHRAE 169-2013-6B', 'ASHRAE 169-2013-7A', 'ASHRAE 169-2013-8A']
+
+  GEOMETRY_FILES = ['Existing Geometry',
+                    'ASHRAESmallOffice.osm',
+                    'ASHRAESmallHotel.osm',
+                    'ASHRAESecondarySchool.osm',
+                    'ASHRAERetailStripmall.osm',
+                    'ASHRAEQuickServiceRestaurant.osm',
+                    'ASHRAEPrimarySchool.osm',
+                    'ASHRAEOutpatient.osm',
+                    'ASHRAEMidriseApartment.osm',
+                    'ASHRAEMediumOffice.osm',
+                    'ASHRAELargeOffice.osm',
+                    'ASHRAELargeHotel.osm',
+                    'ASHRAELaboratory.osm',
+                    'ASHRAEHospital.osm',
+                    'ASHRAEHighriseApartment.osm',
+                    'ASHRAEFullServiceRestaurant.osm',
+                    'ASHRAECourthouse.osm',
+                    'ASHRAECollege.osm',
+
+  ]
```

Comment:
Instead of having a straight dictionary could we perhaps just have "typical geomtry names". For example instead of "ASHRAESmallOffice.osm", could we have "Small Office"? I'm afraid that users won't have much clue as to what "ASHRAESmallOffice.osm" corresponds to, I feel like it's a bit more clear if we remove the ".osm" and the reference to ASHRAE.

Reply:
Actually, this is already the case.